### DISCUSSION TITLE
Inactive window notifications, a couple suggestions

### DISCUSSION THREAD

**Discussion by Dimmitsaras** [ppy/osu#23567] [Ideas]
There should be a way for users to have sound effect volume up but music volume down when the window is unfocused. 
This could be useful for notification purposes (for example a friend messages you or invites you to a multiplayer lobby).

This could also be paired with another change; currently you do not receive a notification (visual or sound) if your chat is open with the notifying party. If you have chat closed or are chatting with another person, this shows up as normal:
![image](https://github.com/ppy/osu/assets/35996393/f5bc1aa5-741f-4bb1-aeda-70e27c638936)

My suggestion is that this should always show (and play the accompanying sound) if the window is inactive.

**Comment by peppy** [comment]
Also we'd want to flash the taskbar, like stable does.
